app/services/migration/migrators/user.rb

```diff
@@ -33,11 +33,21 @@ def call
         trns = unique_validated_trns(ecf_user)
         validate_multiple_trns!(trns, user)
 
+        emails = unique_emails(ecf_user)
+        validate_multiple_emails!(emails, user)
+        validate_existing_email_match!(emails.first, user)
+
+        npq_application = most_recent_updated_npq_application(ecf_user)
+
         user.update!(
           trn: ecf_user.teacher_profile&.trn ? ecf_user.teacher_profile.trn : trns.last,
           full_name: ecf_user.full_name || user.full_name,
-          email: ecf_user.email || user.email,
+          email: emails.first || user.email,
           uid: ecf_user.get_an_identity_id || user.uid,
+          date_of_birth: npq_application&.date_of_birth || user.date_of_birth,
+          national_insurance_number: npq_application&.nino || user.national_insurance_number,
+          active_alert: npq_application&.active_alert,
+          trn_verified: npq_application&.teacher_reference_number_verified,
```

Comment:
in the report we are getting `  PG::NotNullViolation: ERROR:  null value in column "trn_verified" of relation "users" violates not-null constraint`, could we fallback to the user value for both the alert and verified values incase?